cmd/hub/lifecycle/deploy.go

```diff
@@ -684,8 +685,9 @@ func fireHooks(trigger string, stackBaseDir string, component *manifest.Componen
 		}
 		stdout, stderr, err := delegateHook(script, stackBaseDir, component, componentParameters, osEnv)
 		if err != nil {
-			if strings.Contains(err.Error(), "fork/exec : no such file or directory") {
-				log.Printf("Error: file %s has not been found.", script)
+			// script found but can't be executed because of permissions
+			if strings.Contains(err.Error(), "permission denied") {
+				log.Printf("Error: Permission denied. Try chmod +x %s", script)
```

Comment:
Same comment as above. 
Error message might be misleading, because 'script' may not return exact script. See `probeScript` implementation. Therefore I would suggest fail with "permission denied (file not executable)" or something similar but more user friendly

Also "chmod" may confuse Windows users

Reply:
A comment to @akranga  ... `Fatalf` is inappropriate here as will force `system.exit(1)`. Dropping first comment

Reply:
Changed the error

Reply:
I propose a message like: `error: Permission denied. Hook script "%s" should be executable`